requirementsgui.txt

```diff
@@ -0,0 +1,2 @@
+PyQt5==5.12.1
+PyQt5-stubs==5.12.1
```

Comment:
the `PyQt5-stubs` is not needed to run the code, but it makes life way easier when developing.

Reply:
Is there a way we can make a different `make` target for dev instead of having it in the regular user's build?

Reply:
I added `guidev` to the make file and removed PyQt5-stubs from the `gui` make.